rskj-core/src/main/java/co/rsk/net/discovery/NodeChallengeManager.java

```diff
@@ -29,9 +31,15 @@
  * Created by mario on 22/02/17.
  */
 public class NodeChallengeManager {
+    private static final Logger logger = LoggerFactory.getLogger(NodeChallengeManager.class);
+
     private Map<String, NodeChallenge> activeChallenges = new ConcurrentHashMap<>();
 
     public NodeChallenge startChallenge(Node challengedNode, Node challenger, PeerExplorer explorer) {
+        logger.debug("co.rsk.net.discovery.NodeChallengeManager.startChallenge - Starting challenge for node:\n" +
```

Comment:
```suggestion
        logger.debug("startChallenge - Starting challenge for node: [{}] by challenger: [{}]" +
```
I believe that a full path of a class is already included by a logger. This will simply duplicated it, won't it?

Also I'd use the technic used in other places, like `"<something to log>: [{}]"` and avoid new lines (`"\n"`) as some tools might treat it as two separate logs (eg Grafana)

Reply:
I see, I'll remove the path and use the format you mentioned for the variables in the logs message.